File: apps/s-2-admin/src/api/destinationCalendar/DestinationCalendarWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { StringFilter } from "../../util/StringFilter";
import { UserWhereUniqueInput } from "../user/UserWhereUniqueInput";
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";
import { EventTypeWhereUniqueInput } from "../eventType/EventTypeWhereUniqueInput";
import { CredentialWhereUniqueInput } from "../credential/CredentialWhereUniqueInput";

export type DestinationCalendarWhereInput = {
  id?: IntFilter;
  integration?: StringFilter;
  externalId?: StringFilter;
  user?: UserWhereUniqueInput;
  booking?: BookingWhereUniqueInput;
  eventType?: EventTypeWhereUniqueInput;
  credential?: CredentialWhereUniqueInput;
};

File: apps/cactus-demo-dotnet/src/APIs/DestinationCalendar/Dtos/DestinationCalendarWhereInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class DestinationCalendarWhereInput
{
    public int? Id { get; set; }

    public string? Integration { get; set; }

    public string? ExternalId { get; set; }

    public int? User { get; set; }

    public int? Booking { get; set; }

    public int? EventType { get; set; }

    public int? Credential { get; set; }
}
